README.md

```diff
@@ -0,0 +1,52 @@
+# _Travel Journal_
+
+A travel journal showing highlights of locations I've travelled to.
+
+---
+
+## Authors
+
+Kortney Foley
+
+---
+
+## Technologies Used
+
+-   HTML
+-   JavaScript
+-   CSS
+-   React
+-   [TypeScript](https://www.typescriptlang.org/)
+
+---
+
+## Description
+
+The purpose of this app is to display places I've travelled to with key details about each visit, incuding a photo of the location, a link to view the location on Google Maps, the date visited and a short description of what the location is. Using the form at the bottom of the screen, users can create a new journal entry that is saved in local storage.
+
+I designed this app to get better at using props in React and added TypeScript to practise that as well.
+
+### TO DO
+
+-   add back-end so new journal entries aren't just stored locally
+-
```

Comment:
you have a second dash here. did you want to add another todo? or just remove it?